packages/@webex/common/src/browser-detection.js

```diff
@@ -1,6 +1,20 @@
 import bowser from 'bowser';
 import {memoize} from 'lodash';
 import window from 'global/window';
+import {browserDetection} from './constants';
+
+export const getBrowserSerial = () => {
+  let browserData;
+  try {
+    browserData = window?.navigator?.userAgent
```

Comment:
question: instead of doing it this way, why couldn't we just use the existing detection object and have a standard value that appears?

Reply:
Because there is something going wrong when the detection object is created. It's returning null but we don't know why. The bowser plugin might be failing, navigator.userAgent might not be defined, we don't know. We want to narrow the scope so debugging is a little easier.

Reply:
tracks